core/Symbols.cc

```diff
@@ -448,9 +448,16 @@ string TypeMemberRef::show(const GlobalState &gs) const {
     if (sym->name == core::Names::Constants::AttachedClass()) {
         auto attached = sym->owner.asClassOrModuleRef().data(gs)->attachedClass(gs);
         ENFORCE(attached.exists());
-        return fmt::format("T.attached_class (of {})", attached.show(gs));
+        // TODO: This change is needed for RBI generation.
+        return "T.attached_class";
+        // return fmt::format("T.attached_class (of {})", attached.show(gs));
```

Comment:
Migrating @jez's comment forward from `ClassOrModuleRef::show`:

This makes error messages substantially more confusing. We should make separate printing methods before landing this so that we don't have to regress this error message.